packages/discord.js/src/errors/Messages.js

```diff
@@ -113,6 +113,7 @@ const Messages = {
 
   EMOJI_TYPE: 'Emoji must be a string or GuildEmoji/ReactionEmoji',
   EMOJI_MANAGED: 'Emoji is managed and has no Author.',
+  EMOJI_RESOLVE: 'Could not resolve the emoji.',
```

Comment:
This isn’t needed, you can use INVALID_TYPE instead

Reply:
You're right, I changed it, thanks for pointing it out